packages/node_modules/nav-frontend-stegindikator/src/stegindikator.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+
+import StegindikatorSteg, { StegindikatorStegProps } from './stegindikator-steg';
+import { omit } from 'nav-frontend-js-utils';
+
+import 'nav-frontend-stegindikator-style';
+
+const cls = (state) => cn('stegindikator', {
+    'stegindikator--kompakt': state.kompakt
+});
+
+export interface StegindikatorProps {
+    steg: StegindikatorStegProps[];
+    visLabel: boolean;
+    kompakt: boolean;
+    aktivtSteg?: number;
+    onChange?: (index: number) => void;
+    onBeforeChange?: (index: number) => boolean;
```

Comment:
Hva er behovet for en `onBeforeChange` ?

Reply:
Som vi diskutert ang. bruk av `disabled` eller ikke, og at `onChange` muligens vil medføre "flikking" hvis bruker klikker på noe som ikke er tillatt, så kan `onBeforeChange` benyttes for å unngå dette.